### PR TITLE
add support for authz APIs

### PR DESCRIPTION
**Pull request by slavikm**
## Description

Adds Descope Authz APIs to the Node management SDK

## Must

- [ ] Tests
- [ ] Documentation (if applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -840,6 +841,185 @@ const audits = await descopeClient.management.audit.search({ actions: ['LoginSuc
 console.log(audits);
 ```
 
+### Manage Authz
+
+Descope support full relation based access control (ReBAC) using a zanzibar like schema and operations.
+A schema is comprized of namespaces (entities like documents, folders, orgs, etc.) and each namespace has relation definitions to define relations.
+Each relation definition can be simple (either you have it or not) or complex (union of nodes).
+
+A simple example for a file system like schema would be:
```
Comment (asafshen) [inline README.md:850]: just making sure I understand this sentence (and the context of the yaml file bellow) - the yaml file doesn't have any usage from the developer perspective - it is just a readable presentation

File: lib/management/types.ts [hunk 2]
```diff
@@ -240,3 +240,104 @@ export enum UserStatus {
   disabled = 'disabled',
   invited = 'invited',
 }
+
+export enum AuthzNodeExpressionType {
```
Comment (asafshen) [inline lib/management/types.ts:244]: not critical, but note that *TS union string* is consider to be simpler form of enum (unless specific enum feature is required)
e.g for enum string:
image


enums are great as well :)

**Review by asafshen — APPROVED** [review]
excellent work

File: lib/management/types.ts (hunk 2 above)
Comment (slavikm) [reply lib/management/types.ts:244]: Leaving as is because don't want to change `UserStatus` that was there already...

**Pull request merged**